cpp/tests/partitioning/partition_test.cpp

```diff
@@ -30,7 +30,8 @@ class PartitionTest : public cudf::test::BaseFixture {
   using map_type   = cudf::test::GetType<T, 1>;
 };
 
-using types = cudf::test::CrossProduct<cudf::test::FixedWidthTypes, cudf::test::IntegralTypes>;
+using types =
+  cudf::test::CrossProduct<cudf::test::FixedWidthTypes, cudf::test::IntegralTypesNotBool>;
```

Comment:
Does this test not work with `bool`?

Reply:
Yes, the partition gtests fail if you include the `bool` type here. There is logic in the partition code that throws an exception if a `bool` type column is passed:
https://github.com/rapidsai/cudf/blob/b32e777df9f00a7c9e89c701c87fba09f7622f01/cpp/src/partitioning/partitioning.cu#L711-L721

Reply:
Ah, it's used as a map. Okay, cool.